lib/ichabod/resource_set/resource.rb

```diff
@@ -10,7 +10,7 @@ class Resource
       NYUCORE_ATTRIBUTES = [:identifier, :addinfolink, :addinfotext, :available,
```

Comment:
I'd like to propose we create a "metadata terms audit" chore. I want to sit down with some folks in KADD and review the Nyucore Attributes in use and see if we can map them to common vocabularies using RDF.rb.